demo/kubernetes/rook-storageclass.yaml

```diff
@@ -1,3 +1,18 @@
+apiVersion: rook.io/v1alpha1
+kind: Rookpool
+metadata:
+  name: rook-replicapool
+spec:
+  name: replicapool
```

Comment:
there is currently only one operator that controls all the tprs. if we wanted to control the tpr at the cluster level, we would need a cluster-scoped operator. that may make sense for pools and other cluster-specific tprs, though it's not planned